### PR TITLE
Akka Paradox theme 0.25 (with transitive Paradox, apidoc, project-info)

### PR DESCRIPTION
**Pull request by ennru**
## Purpose

Upgrade to theme 0.25 with improved API link features and styling

## Changes

* Depend transitively on Paradox, apidoc, project-info (might reduce Scala Steward-pressure on Akka)

### CODE REVIEW

File: project/plugins.sbt [hunk 1]
```diff
@@ -15,10 +15,7 @@ addSbtPlugin("pl.project13.scala" % "sbt-jmh" % "0.3.7")
 addSbtPlugin("com.typesafe.sbt" % "sbt-native-packager" % "1.4.1")
 addSbtPlugin("io.spray" % "sbt-boilerplate" % "0.6.1")
 addSbtPlugin("com.timushev.sbt" % "sbt-updates" % "0.4.2")
-addSbtPlugin("com.lightbend.akka" % "sbt-paradox-akka" % "0.24")
-addSbtPlugin("com.lightbend.paradox" % "sbt-paradox-apidoc" % "0.3")
-addSbtPlugin("com.lightbend.paradox" % "sbt-paradox-project-info" % "1.1.3")
-addSbtPlugin("com.lightbend.paradox" % "sbt-paradox" % "0.6.5")
+addSbtPlugin("com.lightbend.akka" % "sbt-paradox-akka" % "0.25")
```
Comment (helena) [inline project/plugins.sbt]: 👍

**Review by helena — APPROVED** [review]
LGTM

File: project/plugins.sbt (hunk 1 above)
Comment (patriknw) [reply project/plugins.sbt]: ok, so this means that whenever there is a release of apidoc and project-info there will be a new release of paradox-akka.

**Review by patriknw — APPROVED** [review]
LGTM

**Comment by ennru** [comment]
Yes, my thinking is that we fully control when akka-paradox is released and can choose to pull it in when it fits, and we want the changes in. Furthermore, we get the same combination of those versions across all projects.

**Pull request merged**